inst/templates/CMakeLists-src.txt

```diff
@@ -0,0 +1,20 @@
+add_library({{{project}}}
+ *.c
+ *.h
+ *.cpp
```

Comment:
@krlmlr Like this? or we need to get the list of files from R and display them here?